dev-lua/lua-resty-core/lua-resty-core-0.1.30.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+DESCRIPTION="FFI-based API for lua-nginx-module"
+HOMEPAGE="https://github.com/openresty/lua-resty-core"
+
+SRC_URI="
+	https://github.com/openresty/lua-resty-core/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+"
+LICENSE="BSD-2"
+
+SLOT=0
```

Comment:
fix these blocks please

Reply:
Please fix the whitespace in all the files, and quote SLOT="0". 

It should be like:
```
EAPI=8

DESCRIPTION="..."
HOMEPAGE="..."
SRC_URI="..."

LICENSE=..."
SLOT="0"
```

and so on.